_includes/header/header-en.html

```diff
@@ -134,6 +134,9 @@
                           <li>
                             <a href="/{{ page.lang }}/resources/contributing.html">Contributing to Express</a>
                           </li>
+                          <li>
+                            <a href="/{{ page.lang }}/resources/contributing-docs.html">Contributing to Docs</a>
```

Comment:
please, add also for other languages

Reply:
Do we put it in even tho there is no translation?

Reply:
we want is to maintain consistency across all languages. I know it's complicated, but we're trying. Adding the link and redirecting to the English page would help maintain this consistency, although we probably also want to add the content to its folder even if it isn't translated. However, this would be done last after all the feedback is received

Reply:
I added all the links to en for now.

I personally don't agree with adding untranslated pages into other language folders just because it's quite confusing on our end (and seems wrong), _but_ I do see where you are coming from - adding only the links to `/en` takes users outside their langs, with only the back arrow to get back to it.